app/src/main/res/layout/preference_credit_cards_add_credit_card.xml

```diff
@@ -1,40 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- This Source Code Form is subject to the terms of the Mozilla Public
-   - License, v. 2.0. If a copy of the MPL was not distributed with this
-   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
-    android:layout_width="match_parent"
-    android:layout_height="@dimen/credit_cards_add_credit_card_button_height"
-    android:background="?android:selectableItemBackground"
-    android:clickable="true"
-    android:focusable="true"
-    android:paddingStart="@dimen/top_bar_alignment_margin_start"
-    tools:ignore="RtlSymmetry">
-
-    <ImageView
-        android:id="@+id/add_icon"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:importantForAccessibility="no"
-        android:textAlignment="textStart"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
-        app:srcCompat="@drawable/ic_new" />
-
-    <TextView
-        android:id="@+id/add_credit_card_text"
-        android:layout_width="0dp"
-        android:layout_height="wrap_content"
-        android:layout_marginStart="@dimen/credit_cards_add_credit_card_text_horizontal_margin"
```

Comment:
Can we also remove this unused variable as well? 

`credit_cards_add_credit_card_text_horizontal_margin`,
`credit_cards_add_credit_card_text_size`